Please fix the compiling error of windows

```
 C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\xutility(1866): warning C4503: 'std::allocator<_Other>::destroy' : decorated name length exceeded, name was truncated [f:\win32_slave\workspace\ccs-pr\node\win32_bak\cocos\2d\libcocos2d.vcxproj]


       "f:\win32_slave\workspace\ccs-pr\node\win32_bak\build\cocos2d-win32.sln" (Build target) (1) ->
       "f:\win32_slave\workspace\ccs-pr\node\win32_bak\cocos\2d\libcocos2d.vcxproj" (default target) (5) ->
       (ClCompile target) -> 
         ..\ui\UIPageViewIndicator.cpp(36): error C2146: syntax error : missing ';' before identifier 'GLubyte' [f:\win32_slave\workspace\ccs-pr\node\win32_bak\cocos\2d\libcocos2d.vcxproj]
```

--- a/cocos/ui/UIPageViewIndicator.cpp
+++ b/cocos/ui/UIPageViewIndicator.cpp
@@ -33,7 +33,7 @@ NS_CC_BEGIN
 
 namespace {
     static const float SPACE_BETWEEN_INDEX_NODES_DEFAULT = 23;
-    constexpr GLubyte INDEX_NODES_OPACITY_DEFAULT = 0.3*255;
+    static const GLubyte INDEX_NODES_OPACITY_DEFAULT = 0.3*255;
 }
 
 namespace ui {

